docs/content/troubleshooting/csi_ug_troubleshooting_detect_errors.md

```diff
@@ -9,7 +9,7 @@ This is an overview of actions that you can take to pinpoint a potential cause f
 1.  Verify that the CSI driver is running. (Make sure the `csi-controller` pod status is _Running_).
 
     ```
-    $> kubectl get all -n <namespace> -l csi
+    kubectl get all -n <namespace> -l csi
```

Comment:
we already have this command in the [logs collection page](https://github.com/IBM/ibm-block-csi-driver/blob/8d18ae5a6d95b15894c814e59a27c133aea06d8a/docs/content/troubleshooting/csi_ug_troubleshooting_logs.md), and I don't see why this page is dedicated specifically for the controller, nor why we need this separate page

Reply:
@oriyarde  this page and command isn't specifically about log collection - it's to see the pod status and what to do if it isn't running. This is basic troubleshooting and should be left in.

Reply:
before the actual steps, there is nothing in this page which says that this is specifically about the controller.
and I don't see why we want a separate page specifically for the controller.
IMO this page can be combined with the "log" (+other info) collection page, while generalizing the controller-specific instructions

Reply:
DONE

Reply:
thanks, though I would also generalize the controller-specific instructions. will open a new conversation for it